render_blender.py

```diff
@@ -26,149 +30,209 @@
                     help='Number of bit per channel used for output. Either 8 or 16.')
 parser.add_argument('--format', type=str, default='PNG',
                     help='Format of files generated. Either PNG or OPEN_EXR')
+parser.add_argument('--resolution', type=int, default=600,
+                    help='Resolution of the images.')
 
 argv = sys.argv[sys.argv.index("--") + 1:]
 args = parser.parse_args(argv)
 
-import bpy
+# Set up rendering
+context = bpy.context
+scene = bpy.context.scene
+render = bpy.context.scene.render
+
+render.engine = 'CYCLES' # ('CYCLES', 'BLENDER_EEVEE', ...)
+render.image_settings.color_mode = 'RGBA' # ('RGB', 'RGBA', ...)
+render.image_settings.color_depth = args.color_depth # ('8', '16')
+render.image_settings.file_format = args.format # ('PNG', 'OPEN_EXR', 'JPEG, ...)
+render.resolution_x = args.resolution
+render.resolution_y = args.resolution
+render.resolution_percentage = 100
```

Comment:
Necessary?

Reply:
Not sure what exactly you mean... Always square is better than fixed and always square. Smaller resolution renders faster when playing around with the code.

Reply:
I see, so you're saying that's an easy way (apart from the resolution argument), do adapt the size of the output. Correct?

Can you put that into a code comment here?